Jenkinsfile

```diff
@@ -104,14 +129,20 @@ pipeline {
 				}
 			}
 		}
-		// This stage archives all logs and database backups produced by UpdateStableIdentifiers
+		// This stage archives all logs and database backups produced by UpdateStableIdentifiers in the Reactome s3 bucket.
```

Comment:
Nitpick :) - the 's' in 's3' should be capitalized.  Your serve, sir...

Reply:
:/

Reply:
Oh, I'll get you yet!